idea/src/org/jetbrains/kotlin/idea/inspections/collections/RenameUselessCallFix.kt

```diff
@@ -35,6 +34,25 @@ class RenameUselessCallFix(val newName: String) : LocalQuickFix {
             val selectorCallExpression = it.selectorExpression as? KtCallExpression
             val calleeExpression = selectorCallExpression?.calleeExpression ?: return
             calleeExpression.replaced(factory.createExpression(newName))
+            selectorCallExpression.applyFix(factory, calleeExpression.text, newName)
+        }
+    }
+
+    private fun KtCallExpression.applyFix(factory: KtPsiFactory, calleeName: String, newName: String) {
```

Comment:
This name does not correspond to the executed actions. I'd suggest a name like `renameGivenReturnLabels` and also rename `calleeName` to `labelName`.